src/docs/self-hosted/index.mdx

```diff
@@ -10,9 +10,25 @@ Our recommendation is to download the [latest release of the onpremise reposito
+You are very likely to want to change the default configuration for Sentry. Here are the main facilities available to you for this purpose:
 
-**Once you change your configuration, you'll need to restart all Sentry services by running `docker-compose restart web worker cron sentry-cleanup` (or just `docker-compose restart` to restart everything).**
+1. Most, if not all, configuration options that you'd like to tweak are in `sentry/config.yml`, which is generated from [`sentry/config.example.yml`](https://github.com/getsentry/onpremise/blob/master/sentry/config.example.yml) at the time of installation. The file itself documents the most common configuration options as code comments, and you can find more about configuring Sentry over at <Link to="/config/">the configuration section of our developer documentation</Link>. You would likely need to change `system.url-prefix` (this is also prompted at the welcome screen, right after the installation) and settings starting with `mail.` (though we do ship with a basic SMTP server).  GitHub and Slack integration settings also reside in this file.
+
+1. For more advanced things, you can have a look at [`sentry/sentry.conf.example.py`](https://github.com/getsentry/onpremise/blob/master/sentry/sentry.conf.example.py).
+
+1. There are a few places where we depend on environment variables. The keys are defined in [.env](https://github.com/getsentry/onpremise/blob/master/.env), though if you need to override any of them we expect you to use some other means of setting environment variables dependent on your system.
+
+1. <Link to="/self-hosted/geolocation/">Geolocation</Link> uses a custom configuration file to conform to the underlying technology.
+
+<Alert title="Note" level="info">
+  Once you change your configuration, you'll need to restart all Sentry services by running <code>docker-compose restart web worker cron sentry-cleanup</code> (or just <code>docker-compose restart</code> to restart everything).
+</Alert>
+
+Here is further information on specific configuration topics:
+
+- <Link to="/self-hosted/email/">Email</Link>
+- <Link to="/self-hosted/geolocation/">Geolocation</Link>
+- <Link to="/self-hosted/sso/">Single Sign-On (SSO)</Link>
```

Comment:
You can use Markdown for these links, from our Tips to Contributors, here's how we link link to another page in docs:
1.  Put the linking text inside brackets [ ]
2. After the brackets, put the file path inside ()

For example:
[PagerDuty](/product/integrations/pagerduty/)

Reply:
Ah! I used Markdown syntax initially, but I had a relative URL (`./geolocation`), and gatsby was erroring out. I've confirmed that it works with an absolute URL. 👍